### DISCUSSION TITLE
Proper way to change font inside presetThemeDefault?

### DISCUSSION THREAD

**Discussion by sandros94** [jd-solanki/anu#158] [Q&A] [answered]
I noticed that the scss from the presetThemeDefault applies the `Inter` font:
https://github.com/jd-solanki/anu/blob/d1e14de50348b0572f80d7f8ff566c411a71038c/packages/preset-theme-default/src/scss/index.scss#L9-L11
https://github.com/jd-solanki/anu/blob/d1e14de50348b0572f80d7f8ff566c411a71038c/packages/preset-theme-default/src/scss/index.scss#L89-L107

Can I change it from `uno.config.ts` or should I just overwrite the body and heading styles?

**Comment by sandros94** [comment]
Oh and the color too. (those .68 and .87)

**Reply by jd-solanki** [reply, marked as answer]
Hi @Sandros94 

Thanks for reporting, Actually we were providing SASS file but it looks like we aren't providing it in the latest release.

Moreover, we also have to explore using webfontloader preset. Let's convert this into the issue so I can track and fix it in the next release.

**Reply by sandros94** [reply]
Ok in the meantime I'm closing this discussion

**Reply by jd-solanki** [reply]
Oh, great!

Thanks, You must be nice manager 😇
